gp_emulator/gpu/tests/testCdist.cu

```diff
@@ -0,0 +1,42 @@
+#include "gpu_predict_test.h"
+#define IDX2D(i,j,ld) (((j)*(ld))+(i))
+
+
+void testCdist( const real *in1,const real *in2, const real *res, const int in1_nrows, const int in2_nrows, const int in_ncols )
```

Comment:
Convert these tests to use the Catch C++ testing framework. http://development.rc.ucl.ac.uk/training/rcwithcpp/session01/04UnitTesting.html